Update to 2.023.3

isuruf opened the pull request:
cc @saraedum

isuruf committed:
Update to 1.023

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

isuruf commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.5 and pinning 2018.05.22

saraedum committed:
Simplify build.sh

The readme at https://gitlab.com/rezozer/forks/sympow suggests that this is all that is necessary now

saraedum committed:
Upgrade to 2.023.3

isuruf commented on recipe/meta.yaml:
```diff
@@ -1,23 +1,25 @@
 source:
-  fn: {{ name }}-{{ version }}.tar.bz2
-  url: http://mirrors.mit.edu/sage/spkg/upstream/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  url:  https://salsa.debian.org/science-team/sympow/-/archive/upstream/{{ version }}/sympow-upstream-{{ version }}.tar.gz
```
Is this under the same license? https://salsa.debian.org/science-team/sympow/blob/upstream/README.md#forked-code has only a copyright.

saraedum replied:
yes, it's the same adhoc license as before. just the copyright has changed.

saraedum committed:
Fix sha256

saraedum committed:
sympow now needs autotools

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict.
Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the @ notation in a comment) if you believe this is a bug.

saraedum committed:
Merge remote-tracking branch 'origin/master' into update

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

isuruf committed:
Add help2man

saraedum committed:
OSX does not know about install -t

Upstream MR is here: https://gitlab.com/rezozer/forks/sympow/merge_requests/1

saraedum committed:
fix typo

saraedum committed:
OSX's cp does not know about -t

saraedum commented:
@isuruf: This looks good to me now. There are some warnings during the testing
```
**WARNING** failed to create data bin package cache folder /home/conda/feedstock_root/build_artifacts/sympow_1533253022288/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/share/cache/sympow/datafiles/le64
```
I have not really checked what is going on here but sympow appears to work fine (for Sage's purposes at least.)

isuruf commented:
conda doesn't keep around empty directories and mkdir fails because the parent directories are not there.

isuruf commented:
What do you think about doing something like `touch $PREFIX/share/cache/sympow/datafiles/`

isuruf commented:
```
mkdir -p $PREFIX/share/cache/sympow/datafiles/
touch $PREFIX/share/cache/sympow/datafiles/empty
```

saraedum commented:
Sure, would you mind touching `.keep` instead?

isuruf committed:
Update build.sh
